### ISSUE TITLE
Worked only once Now fails

### ISSUE THREAD

**Issue by br0ken** [Tsume/Maple-Tree#42]
It worked once. I updated Breath of the wild. When it began to fail I deleted it, and tried a fresh download. Same issue. So, since I saw the options I chose had not changed, I even hunted through my registry and deleted all references to it. Still fails. I have an attached screen shot if it helps and the log file is available if you want it instead.
![mapleseedfail](https://cloud.githubusercontent.com/assets/27357931/24962243/fcafb67a-1f60-11e7-959b-6006647f71bf.JPG)

**Comment by Tsume** [comment]
Did you read the error? lol. It's pretty plain and simple. Access denied, you don't have access to that path. I can't really give you much more info then that.

**Comment by Tsume** [comment]
I'm not sure what else I could do to make selecting paths more clear. If you have any ideas, I'm all ears.

Try the version posted at the end of this thread.
https://github.com/Tsume/Maple-Tree/issues/10

It won't fix your specific problem, but it should let you change the settings to something more... correct.

**Comment by br0ken** [comment]
I fixed it. I used an older version to clear a cache I didn't know about.
Thanks

**Comment by br0ken** [comment]
My apologies as well. I only began this wild wii u emulator endeavor yesterday. I used the app one time, it worked perfectly. Then today Zelda breath of the wild dropped an update. So I went to use your program again. That's when I got this error, that may be obvious to you, yet had a complete novice such as myself lost. Then it dawned on me to use an older version. In it I see "clear cache" and is showed me the address of said cache. Once I deleted it, the content version you have began working again. I can't/don't understand it, but either way, it works perfectly now. Thank you for your response and your hard work. My apologies for any inconvenience.
